### PR TITLE
Update consistency02.md

### PR DESCRIPTION
**Pull request by matsduf**
Update consistency02.md

* Updated to new format.
* Added explicit messages.
* Removed copyright statement.
* Removed description/headline from identifier line.

Changes compared to current implementation (develop branch pre v2018.1): **None**

### CODE REVIEW

File: docs/specifications/tests/Consistency-TP/consistency02.md [hunk 1]
```diff
@@ -1,52 +1,77 @@
...
+**CONSISTENCY02**
 
-### Objective
+## Objective
 
 All authoritative name servers must serve the same SOA record (section
-4.2.1) of [RFC 1034](https://tools.ietf.org/html/rfc1034) for the
-tested domain. As per section 3.3.13 of [RFC 1035](https://tools.ietf.org/html/rfc1035),
+4.2.1) of [RFC 1034] for the
+tested domain. As per section 3.3.13 of [RFC 1035]
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency02.md]: Have you considered using `fmt` or a similar tool to manage the line width? `fmt` is far from perfect but I use it a lot.

Comment (matsduf) [reply docs/specifications/tests/Consistency-TP/consistency02.md]: No. -- It was on purpose that I did not change the line breaks to make it easier to compare.

**Comment by matsduf** [comment]
@mattias-p, please review again.

**Comment by matsduf** [comment]
@sandoche2k, can you please review this PR?

**Pull request merged**